qiskit/optimization/converters/__init__.py

```diff
@@ -21,6 +21,15 @@
 This is selection of converters having encode, decode functionality to go between different
 forms.
```

Comment:
```suggestion
This is a set of converters having `convert` functionality to go between different representations of a given
:class:`~qiskit.optimization.problems.QuadraticProgram` and to `interpret` a given result for the problem,
based on the original problem before conversion, to return an appropriate
:class:`~qiskit.optimization.algorithms.OptimizationResult`.
```
Since the encode and decode is deprecated, maybe this text should be updated and improved to explain better what these do.